core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -65,7 +67,7 @@ private[deploy] object SparkSubmitAction extends Enumeration {
  * This program handles setting up the classpath with relevant Spark dependencies and provides
  * a layer over the different cluster managers and deploy modes that Spark supports.
  */
-object SparkSubmit extends CommandLineUtils {
+object SparkSubmit extends CommandLineUtils with Logging {
```

Comment:
You can't do this. This breaks the logging configuration of `spark-shell` and other shells (which is WARN by default instead of INFO).